### ISSUE TITLE
[2.0.0-beta-9] Can't edit submission that contains email field with "Unique Value" activated

### ISSUE THREAD

**Issue by mtnorthrop** [verbb/formie#904]
**Description**
When I try to edit a submission that contains an email field with the "Unique Value", I'm unable to save the edited submission unless I _also_ change the contents of the email field.

If I leave the email field unchanged, I'm given a "“Email” must be unique." validation error.

**Steps to reproduce**
1. Create a form with an email field whose "Unique Value" toggle is turned on.
2. Submit the form
3. Open the submission in the control panel and try to save the submission without changing any of the fields
4. You should be shown an "Email must be unique" validation error on the email field.

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: dev-craft-4 as 2.0.0-beta.9
- Craft version: 4.0.3
- Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by mtnorthrop** [comment]
Thanks! (I hope I'm not being too annoying with all these bug reports!)

**Comment by engram-design** [comment]
Not at all! Really appreciate the feedback, especially as we're doing some major updates for v2.

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.10](https://github.com/verbb/formie/releases/tag/2.0.0-beta.10)
